lib/components/molecules/ArticleThumbnail/tests/ArticleThumbnail.test.js

```diff
@@ -0,0 +1,14 @@
+import { shallow } from 'enzyme';
+
+import ArticleThumbnail from '../index';
+
+describe('<ArticleThumbnail />', () => {
+  let ArticleThumbnailComponent = '';
+  beforeEach(() => {
+    ArticleThumbnailComponent = shallow(<ArticleThumbnail>Test</ArticleThumbnail>);
```

Comment:
The test cases should make use of sample data, should do snapshot of all variations.

Reply:
Done